django_project/trail/serializers/point_of_interest.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from ..models.point_of_interest import POI
+
+__author__ = 'Alison Mukoma <alison@kartoza.com>'
+__date__ = '01/10/2018'
+__license__ = "GPL"
+__copyright__ = 'kartoza.com'
+
+
+class POISerializer(serializers.ModelSerializer):
+    """Serializer class for a Point of Interest model."""
+
+    class Meta:
+        model = POI
+        fields = ('name', 'notes', 'guid')
```

Comment:
please try to be consistent (maintain the element of least surprise) - so e.g.

1. guid
2. name
3. image
4. notes (if set)
5.. .... other fields

Do this across all your serialisers and your API users will thank you for a predictable, consistent API...